controller/codebase_blackbox_test.go

```diff
@@ -343,12 +343,13 @@ func (s *CodebaseControllerTestSuite) TestUpdateCodebase() {
 			},
 		}
 	}
-	r, err := recorder.New("../test/data/gemini-scan/codebase-update-ok")
-	require.NoError(t, err)
-	defer r.Stop()
-	m := httpmonitor.NewTransportMonitor(r.Transport)
 
 	t.Run("OK", func(t *testing.T) {
+		r, err := recorder.New("../test/data/gemini-scan/codebase-update-ok")
```

Comment:
Any verification that 'create' does _not_ call gemini?

Reply:
it does not have a way or controller to call gemini.